src/services/api/loan-service.ts

```diff
@@ -0,0 +1,21 @@
+import { baseUrl } from "@/shared/constant";
+
+const createNewLoan = async (loanData: any) => {
+
+    try {
+        const response = await fetch(`${baseUrl}/api/loan`, {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify(loanData),
+        });
+        const newLoan = await response.json();
+        return newLoan;
+    } catch (error) {
+        console.log("Failed creating new loan, api/loan: ", error);
+
+    }
```

Comment:
Return something in case of loan, if is an object return a null

Reply:
noted